util/gpu_t.cuh

```diff
@@ -59,6 +59,7 @@ class stream_t {
 public:
     stream_t(int id) : gpu_id(id)
     {   cudaStreamCreateWithFlags(&stream, cudaStreamNonBlocking);   }
+    stream_t(int id, cudaStream_t _stream) : stream(_stream), gpu_id(id) {};
```

Comment:
The style of {} in this file looks strange. Does it follow our formatter?

Reply:
No, the third-party libraries do not have our format applied.